### ISSUE TITLE
Publish typings for typed-microphone-stream

### ISSUE THREAD

**Issue by sebastianhaas** [opensheetmusicdisplay/opensheetmusicdisplay#14]
https://github.com/sebastianhaas/typed-microphone-stream

Write tests, complete type definition and get it to the typings registry.

**Comment by sebastianhaas** [comment]
@oliver-h would be interested in adding the missing stuff since you are already using it and know the API?

**Comment by oliver-h** [comment]
Yes of course, but for now it's rather a low-prio task for me...

**Comment by sebastianhaas** [comment]
Let's see how this typings thing evolves after all. Microsoft pushes a npm solution as of TS2.0
